spec/API_specification/linear_algebra_functions.md

```diff
@@ -270,9 +270,25 @@ TODO
 TODO
 
 (function-solve)=
-### solve()
+### solve(x1, x2, /)
 
-TODO
+Returns the solution to the system of linear equations represented by the well-determined (i.e., full rank) linear matrix equation `AX = B`.
+
+#### Parameters
+
+-   **x1**: _&lt;array&gt;_
+
+    -   coefficient array `A` having shape `(..., M, M)` and whose innermost two dimensions form square matrices. Must be of full rank (i.e., all rows or, equivalently, columns must be linearly independent). Must have a data type of either `float32` or `float64`.
+
+-   **x2**: _&lt;array&gt;_
+
+    -   ordinate (or "dependent variable") array `B` having shape `(..., M, K)`. Must have a data type of either `float32` or `float64`.
+
+#### Returns
+
+-   **out**: _Tuple\[ &lt;array&gt;, ... ]_
```

Comment:
For `solve()`, can't we just return an array? I don't think we have established the convention to always return (named)tuple in all `array_api.linalg` functions, have we? For example, SVD can return an array if u & v are not required.

Reply:
> I don't think we have established the convention to always return (named)tuple in all `array_api.linalg` functions, have we? 

Maybe this is for the discussion in #95?

> For example, SVD can return an array if u & v are not required.

Perhaps this should be fixed instead. I'll raise in the SVD PR.

Reply:
@leofang I've updated the proposal to only return an `array`. Wrt #95, my current sense is that, if a library wants to return additional info, they can provide a separate API (e.g., `solve_w_info`).